### PR TITLE
responsibleai text and vision 0.0.8 components

### PR DESCRIPTION
**Pull request by imatiach-msft**
responsibleai text and vision 0.0.8 components

### CODE REVIEW

**Review by vinuthakaranth — APPROVED** [review]

File: responsibleai/text/components/rai_text_insights/spec.yaml [hunk 1]
```diff
@@ -1,59 +1,75 @@
...
     enum: ['text_classification', 'multilabel_text_classification']
+    description: The type of task to perform
   model_input: # mlflow model name:version
     type: mlflow_model
     optional: false
+    description: The model to be used for computing insights
   model_info:
     type: string # model name:version
     optional: false
+    description: The model to be used for computing insights
```
Comment (gaugup) [inline responsibleai/text/components/rai_text_insights/spec.yaml:20]: Should this be model name in the description?

File: responsibleai/text/components/rai_text_insights/spec.yaml [hunk 2]
```diff
@@ -1,59 +1,75 @@
...
     default: ''
+    description: Optional text column name, required if metadata present
   maximum_rows_for_test_dataset:
     type: integer
     default: 5000
+    description: The maximum number of rows to use from the test dataset
   classes:
     type: string # Optional[List[str]]
     default: '[]'
+    description: The class names for the target column
```
Comment (gaugup) [inline responsibleai/text/components/rai_text_insights/spec.yaml:39]: Maybe reword to "The list of class names.."

**Review by gaugup — CHANGES_REQUESTED** [review]

File: responsibleai/text/components/rai_text_insights/spec.yaml [hunk 3]
```diff
@@ -1,59 +1,75 @@
...
     default: '[]'
+    description: The list of categorical metadata feature names
   dropped_metadata_features:
     type: string # Optional[List[str]]
     default: '[]'
+    description: The list of dropped metadata feature names
   enable_explanation:
     type: boolean
     default: True
+    description: Whether to enable computing explanations
```
Comment (gaugup) [inline responsibleai/text/components/rai_text_insights/spec.yaml:51]: Maybe reword to "Whether to enable computation of explanations"?

File: responsibleai/text/components/rai_text_insights/spec.yaml [hunk 4]
```diff
@@ -1,59 +1,75 @@
...
   enable_error_analysis:
     type: boolean
     default: True
+    description: Whether to enable error analysis
```
Comment (gaugup) [inline responsibleai/text/components/rai_text_insights/spec.yaml:55]: Maybe reword to "Whether to enable computation of error analysis"?

File: responsibleai/vision/components/rai_vision_insights/spec.yaml [hunk 5]
```diff
@@ -1,54 +1,65 @@
...
-    optional: true
+    description: The type of task to perform
   model_input: # mlflow model name:version
     type: mlflow_model
     optional: false
+    description: The model to be used for computing insights
   model_info:
     type: string # model name:version
     optional: false
+    description: The model to be used for computing insights
```
Comment (gaugup) [inline responsibleai/vision/components/rai_vision_insights/spec.yaml:20]: Should this be model name in the description?

File: responsibleai/vision/components/rai_vision_insights/spec.yaml [hunk 6]
```diff
@@ -1,54 +1,65 @@
...
     type: string
+    description: The target column name
   maximum_rows_for_test_dataset:
     type: integer
     default: 5000
+    description: The maximum number of rows to use from the test dataset
   classes:
     type: string # Optional[List[str]]
     default: '[]'
+    description: The class names for the target column
```
Comment (gaugup) [inline responsibleai/vision/components/rai_vision_insights/spec.yaml:34]: Maybe reword to "The list of class names.."

**Review by gaugup — COMMENTED** [review]
Minor description string changes requested.

File: responsibleai/text/components/rai_text_insights/spec.yaml (hunk 1 above)
Comment (imatiach-msft) [reply responsibleai/text/components/rai_text_insights/spec.yaml]: done!

File: responsibleai/text/components/rai_text_insights/spec.yaml (hunk 3 above)
Comment (imatiach-msft) [reply responsibleai/text/components/rai_text_insights/spec.yaml]: done!

File: responsibleai/text/components/rai_text_insights/spec.yaml (hunk 4 above)
Comment (imatiach-msft) [reply responsibleai/text/components/rai_text_insights/spec.yaml]: done!

File: responsibleai/vision/components/rai_vision_insights/spec.yaml (hunk 5 above)
Comment (imatiach-msft) [reply responsibleai/vision/components/rai_vision_insights/spec.yaml]: done!

File: responsibleai/vision/components/rai_vision_insights/spec.yaml (hunk 6 above)
Comment (imatiach-msft) [reply responsibleai/vision/components/rai_vision_insights/spec.yaml]: done!

**Review by romanlutz — APPROVED** [review]
